server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -398,17 +402,15 @@
             },
             "height": {
               "description": "50373000\tBody height measure [m]",
-              "examples": ["1.55", "0.92", "1.75"],
-              "pattern": "^\\d\\.\\d{2}$",
+              "minimum": 0,
               "title": "Height in m",
-              "type": "string"
+              "type": "number"
             },
             "midUpperArmCircumference": {
               "description": "284473002\tMid upper arm circumference (MUAC) [cm]",
-              "examples": ["23.0", "21.4"],
-              "pattern": "^\\d+\\.\\d+$",
-              "title": "MUAC in cm",
-              "type": "string"
+              "minimum": 0,
+              "title": "MUAC in cm]",
```

Comment:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/9192912/228405291-0c78dd11-5ce6-493d-8793-c8f9bb599e26.png">

We've gained a `]`

Reply:
Eek :) f104bc5df